### ISSUE TITLE
Failed to log in: unexpected connection event from server

### ISSUE THREAD

**Issue by MarkoBL** [mautrix/whatsapp#444]
This morning, I got the message `You're logged in as *** (device #2), but you don't have a WhatsApp connection`. A `reconnect` didn't work, so I issued a `delete-session`. 

And when I tried to login again, I got `Failed to log in: unexpected connection event from server`.

There's nothing in the log file.

**Comment by kumarunster** [comment]
same here

**Comment by Haringstad** [comment]
I can confirm the same here.

```
[Feb 17, 2022 08:34:52] [User/@jacco:matrix.haringstad.com/Client/Socket/ERROR] Error reading from websocket: websocket: close 1006 (abnormal closure): unexpected EOF
[Feb 17, 2022 08:34:52] [User/@jacco:matrix.haringstad.com/Client/Socket/DEBUG] Frame websocket read pump exiting 0xc000365c30
[Feb 17, 2022 08:34:52] [User/@jacco:matrix.haringstad.com/Client/DEBUG] OnDisconnect() called, but it was expected, so not emitting event

```
[Feb 17, 2022 08:34:53] [Puppet/31619354545@s.whatsapp.net/DEBUG] Skipping sync processing: custom user not connected to whatsapp

**Comment by rreuvekamp** [comment]
What version of the bridge are you using? I was on v0.2.2 and upgrading to v0.2.4 helped.

**Comment by MarkoBL** [comment]
I upgraded from 0.2.1 to 0.2.4

**Comment by mrclauss** [comment]
Fixed for me by upgrading mautrix-whatsapp v0.2.3+dev.aacb1d57 (Jan 19 2022, 14:23:50) -> mautrix-whatsapp v0.2.4 (Feb 16 2022, 10:19:14)

**Comment by MarkoBL** [comment]
Doh, I was looking at the wrong server 🤐. Using 0.2.4 fixed the issue for me  👍.

**Comment by fulljackz** [comment]
Fixed here too with latest 0.2.4, thanks !
